### PR TITLE
[ALLUXIO-2441 ] MiniBenchmark

### PR DESCRIPTION
**Pull request by peisun1115**
https://alluxio.atlassian.net/browse/ALLUXIO-2441

A simple tool to run a read/write benchmark. I am not sure whether you guys are ok with checking in this.

### CODE REVIEW

**Comment by peisun1115** [comment]
@calvinjia @apc999

File: examples/src/main/java/alluxio/cli/MiniBenchmark.java [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+   * @throws Exception if error occurs during tests
+   */
+  public static void main(String[] args) throws Exception {
+    if (args.length != 4) {
+      usage();
+      System.exit(-1);
+    }
+    AlluxioURI masterLocation = new AlluxioURI(args[0]);
+    Configuration.set(PropertyKey.MASTER_HOSTNAME, masterLocation.getHost());
+    Configuration.set(PropertyKey.MASTER_RPC_PORT, Integer.toString(masterLocation.getPort()));
```
Comment (apc999) [inline examples/src/main/java/alluxio/cli/MiniBenchmark.java]: NIT: no need to call `Integer.toString()`

File: examples/src/main/java/alluxio/cli/MiniBenchmark.java [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+  }
+
+  private MiniBenchmark() {} // prevent instantiation
+
+  private static void usage() {
+    System.out.println("Usage:");
+    System.out.println("To run a mini benchmark to read or write a file.");
+    System.out.println(String
+        .format("java -cp %s %s <master address> <[READ, WRITE]> <fileSize> <iterations>",
+        RuntimeConstants.ALLUXIO_JAR, TestRunner.class.getCanonicalName()));
```
Comment (apc999) [inline examples/src/main/java/alluxio/cli/MiniBenchmark.java]: `MiniBenchmark` or `TestRunner`?

File: examples/src/main/java/alluxio/cli/MiniBenchmark.java [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+    AlluxioURI masterLocation = new AlluxioURI(args[0]);
+    Configuration.set(PropertyKey.MASTER_HOSTNAME, masterLocation.getHost());
+    Configuration.set(PropertyKey.MASTER_RPC_PORT, Integer.toString(masterLocation.getPort()));
+    ClientContext.init();
+
+    OperationType operationType = Enum.valueOf(OperationType.class, args[1]);
+    long fileSize = FormatUtils.parseSpaceSize(args[2]);
+    int iterations = Integer.parseInt(args[3]);
+
+    switch (operationType) {
```
Comment (apc999) [inline examples/src/main/java/alluxio/cli/MiniBenchmark.java]: shall we run `alluxio.util.CommonUtils#warmUpLoop` to warm up JVM?

File: examples/src/main/java/alluxio/cli/MiniBenchmark.java (hunk 2 above)
Comment (peisun1115) [inline examples/src/main/java/alluxio/cli/MiniBenchmark.java]: done.

File: examples/src/main/java/alluxio/cli/MiniBenchmark.java (hunk 1 above)
Comment (peisun1115) [inline examples/src/main/java/alluxio/cli/MiniBenchmark.java]: done

File: examples/src/main/java/alluxio/cli/MiniBenchmark.java (hunk 3 above)
Comment (peisun1115) [inline examples/src/main/java/alluxio/cli/MiniBenchmark.java]: good point. done

**Comment by peisun1115** [comment]
@apc999  I have addressed your comments. Thanks for the review. PTAL.

**Comment by peisun1115** [comment]
Jenkins, test this please

**Comment by peisun1115** [comment]
@apc999 can you help to merge?.   thanks

**Comment by ifcharming** [comment]
LGTM

**Pull request merged**